ui/admin/app/templates/scopes/scope/host-catalogs/host-catalog/hosts/host/index.hbs

```diff
@@ -17,7 +17,7 @@
   </page.actions>
 
   <page.navigation>
-    <HostCatalogs::HostCatalog::Hosts::Host::Navigation @model={{@model}} />
+    <HostCatalogs::HostCatalog::Hosts::Host::Navigation />
```

Comment:
Apologies, I should have added a comment, I remove this because the `@model` argument is not used in the corresponding component template. If i am mistaken then this change can be removed.